openai/azure-openai/runtime/src/test/java/io/quarkiverse/langchain4j/azure/openai/runtime/AzureOpenAiRecorderEndpointTests.java

```diff
@@ -205,6 +205,11 @@ public Double frequencyPenalty() {
                     return null;
                 }
 
+                @Override
```

Comment:
Shouldn't a test case for the JSON use be added as well ?

Reply:
Testing is for beginners, expert just do it ;)  Well, without joking, for sure a test could be added, but I needed a quick working version and I'm pretty sure that the impact from this change is minimal to the rest of the code.

Reply:
Yes, I'm sure it won't break the rest of the code :)
Just in case, this changes, the code won't be protected :)
If I have the time, I'll try to add a test